docs/use/formatting_outputs.md

```diff
@@ -247,3 +333,5 @@ You can then select the renderer plugin in your `conf.py`:
 ```python
 nb_render_plugin = "default"
 ```
+
+TODO and example of overriding the renderer ...
```

Comment:
We should either add in this example, or make this a comment `% TODO: ...`

Reply:
fixed